components/chainhook-sdk/Cargo.toml

```
@@ -8,11 +8,11 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-serde = { version = "1", features = ["rc"] }
+serde = { version = "1.0.228", features = ["rc"] }
 serde_json = { version = "1", features = ["arbitrary_precision"] }
 serde_derive = "1"
-stacks-codec = "2.4.1"
-clarity = { git = "https://github.com/stacks-network/stacks-core.git", branch = "feat/clarity-wasm-develop", package = "clarity", default-features = false, features = ["log"] }
+stacks-codec = { git = "https://github.com/hirosystems/clarinet.git", version = "3.9.1" }
```

```suggestion
stacks-codec = { git = "https://github.com/stx-labs/clarinet.git", version = "3.9.1" }
```

Both work fine to be fair, github just handles the redirection. But this might be less confusing